### PR TITLE
Add holder for FLN quantization

### PR DESCRIPTION
**Pull request by kawakami-masaki0**
## Pull Request Description:
Add holder for FLN quantization

## Checklist before requesting a review:
- [x] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [x] All function and files are well documented. 
- [x] All function and classes have type hints.
- [x] There is a licenses in all file.
- [x] The function and variable names are informative. 
- [x] I have checked for code duplications.
- [x] I have added new unittest (if necessary).

### CODE REVIEW

File: tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/builder/test_get_activation_quantizer_holder_fln.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    assert isinstance(activation_quantization_holder, PytorchActivationQuantizationHolder)
+    assert activation_quantization_holder.activation_holder_quantizer.num_bits == 8
+    assert activation_quantization_holder.activation_holder_quantizer.signed == False
+    assert activation_quantization_holder.activation_holder_quantizer.threshold_np == 4.0
+
+def test_get_fln_activation_quantizer_holder(fw_impl_mock):
+
+    activation_quantizers = Mock()
+    activation_quantizers.num_bits = 2
```
Comment (kkawa14) [inline tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/builder/test_get_activation_quantizer_holder_fln.py:39]: 同様16bitでの確認

File: tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+# limitations under the License.
+# ==============================================================================
+import model_compression_toolkit as mct
+import torch
+from mct_quantizers import PytorchActivationQuantizationHolder, PytorchFLNActivationQuantizationHolder
+
+from tests_pytest._test_util.tpc_util import configure_mp_activation_opsets
+from model_compression_toolkit.target_platform_capabilities.schema.v2 import QuantizationMethod, AttributeQuantizationConfig, \
+    OpQuantizationConfig, QuantizationConfigOptions, Signedness, OperatorSetNames, TargetPlatformCapabilities, Fusing, OperatorsSet
+from tests.common_tests.helpers.generate_test_tpc import generate_test_attr_configs, generate_test_op_qc
```
Comment (gouda-youichi) [inline tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py:22]: pycharmの警告で、
Access to a protected member _test_util of a module
と出ていますが、問題ないでしょうか。

File: tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+                     OperatorSetNames.RELU,
+                     OperatorSetNames.SIGMOID,
+                     OperatorSetNames.FULLY_CONNECTED,
+                     OperatorSetNames.HARDSWISH],
+        base_op_config=default_op_cfg,
+        a_nbits=[8]
+    )
+    default_cfg = QuantizationConfigOptions(quantization_configurations=[default_op_cfg])
+
+    test_qc = generate_test_op_qc(**generate_test_attr_configs(default_cfg_nbits=8, default_cfg_quantizatiom_method=QuantizationMethod.POWER_OF_TWO))
```
[the remark this answers is not in the document]
Comment (kawakami-masaki0) [reply tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py]: FLNノードは16bitで確認

File: tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/builder/test_get_activation_quantizer_holder_fln.py (hunk 1 above)
Comment (kawakami-masaki0) [reply tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/builder/test_get_activation_quantizer_holder_fln.py]: 同様に、テストにおけるFLNノードは16bitで確認

File: tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py (hunk 2 above)
Comment (kawakami-masaki0) [reply tests_pytest/pytorch_tests/e2e_tests/test_fln_quantization_holder.py:22]: _test_utilフォルダはもともと存在し、他のテストでも使用しているため、問題ないと思います。

**Review by gouda-youichi — APPROVED** [review]

File: tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/test_get_inferable_quantizers.py [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+    _, activation_quantizers = get_inferable_quantizers(node=node_mock, get_weights_quantizer_for_node=None, 
+                                                        get_activations_quantizer_for_node=get_activations_quantizer_for_node_mock)
+    
+    assert len(activation_quantizers) == 1
+    assert isinstance(activation_quantizers[0], ActivationPOTInferableQuantizer)
+    assert activation_quantizers[0].num_bits == 4
+    assert activation_quantizers[0].signed == False
+    assert activation_quantizers[0].threshold_np == 4.0
+
+    # NO_QUANT and FLN_NO_QUANT node
```
[the remark this answers is not in the document]
Comment (kawakami-masaki0) [reply tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/test_get_inferable_quantizers.py:58]: get_inferable_quantizers.pyで修正した条件文がQUANTとFLN_QUANTが対象で
NO_QUANTやFLN_NO_QUANTは対象外で同じ挙動のはずで、まとめてテストしました。

[the remark this answers is not in the document]
Comment (KazunoriSumiya) [reply tests_pytest/pytorch_tests/unit_tests/exporter/model_wrapper/test_get_inferable_quantizers.py:58]: 了解です

**Review by KazunoriSumiya — APPROVED** [review]
